todo.txt

```diff
@@ -0,0 +1,196 @@
+DONE Country of claim
```

Comment:
Probably we don't need this in the master branch, correct?

Reply:
This can now go yes